client/src/Main.elm

```diff
@@ -27,6 +27,7 @@ main =
 type Msg
     = Outgoing Value
     | Incoming String
+    | ChangeLanguage String
```

Comment:
maybe an enum here instead of string ? like

type Languages = Lyng | Slyng | Logic | LogicWithQualifiers

Reply:
I also thought about it, but that would make line 61 significantly more compolex.